---
layout: page
---
<h1 class="projects">    Projects</h1>

<div class="section row">
    <div class="image col-md-4">
        <img class="img-responsive" alt="Project 1 illustration" src="http://placehold.it/250x250"/ Align="Left">
    </div>
    <div class="col-md-8">
        <h3 class="heading">Modeling the impact of genetic variation on multiple types of cellular traits</h3>
        <div class="content">
            We are developing integrative computational approaches for predicting the impact of genetic variation 
            on multiple types of cellular traits, including DNA methylation, histone modification,and gene expression.
            <br>
            <br>
            People: Bernard Ng, Farnush Kiadeh
            <br>
            Alumni: Rosemary McCloskey, James Topham
            <br>
            Relevant publications:
            <br>Battle A, Mostafavi S, Zhu X, Potash JB, et al. Characterizing the genetic basis of transcriptome diversity through RNA-sequencing of 922 individuals. Genome Research (2014)
            <br> Raj T, Rothamel K, Mostafavi S, Ye C, et al. Polarization of the effects of autoimmune and neurodegenerative risk alleles in leukocytes. Science (2014).
        </div>
    </div>
</div>

<div class="section row">
    <div class="image col-md-4">
        <img class="img-responsive" alt="Project 2 illustration" src="http://placehold.it/250x250"/>
    </div>
    <div class="col-md-8">
        <h3 class="heading">Deconvolution of hidden and known confounding factors</h3>
        <div class="content">
            We are developing machine learning methods for modeling the effect of hidden and "partially" confounding factors in gene expression and methylation data. 
            Within this direction, we are also interested in estimating cell type proportions in under-studied mixed tissues such as brain and cord blood.
            <br>
            <br>
            Peaple: Louie Dinh
        </div>
    </div>
</div>
<div class="section row">
    <div class="col-md-4 image">
        <img class="img-responsive" alt="Project 3 illustration" src="http://placehold.it/250x250"/>
    </div>
    <div class="col-md-8">
        <h3 class="heading">Multi omics analysis of complex disease</h3>
        <div class="content">
            Working with clinical colleagues at UBC and elsewhere, we are developing association analysis approaches using multiple types of omics data. 
            Specifically, we are currently focusing on integrative analysis of genotype, gene expression, DNA methylation, and histone modification
            data in order to identify genes and networks that are associated with neurodegeneration and cognifive function.
            <br><br>
            People: Bernard Ng, Halldor Thorhallsson
        </div>
    </div>
</div>
<div class="section row">
    <div class="image col-md-4">
        <img class="img-responsive" alt="Project 4 illustration" src="http://placehold.it/250x250"/>
    </div>
    <div class="col-md-8">
        <h3 class="heading">Constructing tissue- and cell-type specific regulatory networks</h3>
        <div class="content">
            We have developed machine learning approaches for constructing gene functional association and gene regulatory networks in a context- and tissue-specific manner.
            <br><br>
            Alumni: Emma Pierson
        </div>
    </div>
</div>
